schema/helpers/mask.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Mask",
+    "description": "Mask for layer content.",
+    "allOf": [
+        {
+            "properties": {
```

Comment:
Should some of these properties be required? I think `pt` for sure. The rest seem like though could be optional, though I wonder if players will handle that.

Reply:
Required `pt` makes sense, added.